OrbitClientGgp/ClientGgp.cpp

```diff
@@ -55,17 +55,21 @@ bool ClientGgp::InitClient() {
 
 // Client requests to start the capture
 bool ClientGgp::RequestStartCapture(ThreadPool* thread_pool) {
-  ErrorMessageOr<void> result = Capture::StartCapture();
-  if (result.has_error()) {
-    ERROR("Error starting capture: %s", result.error().message());
-    return false;
-  }
   int32_t pid = Capture::GTargetProcess->GetID();
   LOG("Capture pid %d", pid);
 
   // TODO: selected_functions available when UploadSymbols is included
-  std::map<uint64_t, orbit_client_protos::FunctionInfo*> selected_functions =
-      Capture::GSelectedFunctionsMap;
+  // TODO(kuebler): function selection should be handled separately by each
+  //  client.
+  absl::flat_hash_map<uint64_t, orbit_client_protos::FunctionInfo>
+      selected_functions;
+  ErrorMessageOr<void> result = Capture::StartCapture(
+      options_.capture_pid, Capture::GTargetProcess->GetName(),
```

Comment:
```suggestion
      pid, Capture::GTargetProcess->GetName(),
```

Reply:
Options is used in the InitCapture() method to set the process, so here it should be using the pid that is retrieved from that process created and not the value in options

Reply:
I would be fine, if you store the pid/process in the client, but don't want to use the GTargetProcess here.

Reply:
I agree, I can update the code to handle the process locally but right not it is using anyway by using Capture::GTargetProcess->GetName(). That's why I think it makes sense to not update the StartCapture in this step to provide the Process and do that in a separate PR

Reply:
Anyways, will go for GTargetProcess for now, as it is used anyways.

Reply:
I can work to change the ggp client and create a local target process so the other change is easier for you if you remove GTargetProcess in a next step

Reply:
Cool, thanks!

Reply:
Ok I have merged the change, so now you can use target_process_ instead :)